.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,95 @@
+name: Bug report
+description: Submit a bug report
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **New to PcapPlusPlus?**
+
+        For help or advice on using PcapPlusPlus, try one of the following options instead of opening a GitHub issue:
+
+        - Post a message in PcapPlusPlus Google group: https://groups.google.com/d/forum/pcapplusplus-support
+        - Ask a question on Stack Overflow: https://stackoverflow.com/questions/tagged/pcapplusplus
+        - Send an email to: pcapplusplus@gmail.com
+        - Follow us on Twitter: https://twitter.com/seladb
+
+        Make sure to also search the [PcapPlusPlus issue tracker](https://github.com/seladb/PcapPlusPlus/issues) to check that the bug has not already been reported.
+
+        You can check more information about how to report a bug in the [PcapPlusPlus page](https://pcapplusplus.github.io/community#report-an-issue).
+  - type: textarea
+    attributes:
+      label: "Bug description:"
+      description: >
+        Give a clear and concise description of what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        [Copy and paste code where possible rather than using screenshots](https://meta.stackoverflow.com/a/285557/13990016),
+        and put any code blocks inside triple backticks.
+
+      value: |
+        ```cpp
+        # Add a code block here, if required
+        ```
```

Comment:
Should we add a bug report template with specific questions people need to answer to make the bug report more structured?

Reply:
I think we already cover most of the questions that we are interested in, e.g. version, compiler, OS, etc.

Perhaps we can also add a section for backtrace, or error log, but I think these don't have to be a standalone question.

People should be able to add more information in the "description" section.

Do you have specific questions want to add to the template?

Reply:
Currently the bug description only includes a template for C++ code:
![image](https://github.com/seladb/PcapPlusPlus/assets/9059541/844dc063-f6b3-45b0-bca6-432b44016637)

When I first looked at it I thought I only need to fill out the code example, and I searched where can I write the bug description 🙂 

I just opened a bug in Npcap and their template is quite nice, although we need different topics:
https://github.com/nmap/npcap/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=

Maybe:
```markdown
**Describe the bug**
A clear and concise description of what the bug is.

**Code example to reproduce**
(code template)

**Expected behavior**
A clear and concise description of what you expected to happen.
```

Reply:
Okay, I will apply your suggestion, but I would prevent putting too many words inside, most of the time, I also ignore those "templates" when I submit issues in other projects. If we put too much stuff, people will just feel annoyed and ignore it.

Reply:
updated.